storage/innobase/buf/buf0flu.cc

```diff
@@ -2630,36 +2624,34 @@ buf_get_total_LRU_list_length(void)
 /*********************************************************************//**
 Adjust the desired page cleaner thread sleep time for LRU flushes.  */
 MY_ATTRIBUTE((nonnull))
-static
-void
+static void
 page_cleaner_adapt_lru_sleep_time(
-/*==============================*/
-	ulint*	lru_sleep_time,	/*!< in/out: desired page cleaner thread sleep
-				time for LRU flushes  */
-	ulint	lru_n_flushed) /*!< in: number of flushed in previous batch */
-
+    ut_monotonic_time
+	* lru_sleep_time, /*!< in/out: desired page cleaner thread
```

Comment:
Suspicious formatting (just clang-format, there is a format file at storage/innobase in 5.6)

Reply:
this _is_ the result of clang-format, this is why I was asking about formatting because the result doesn't look like the surrounding code.

Reply:
So on the clang-format bits, as discussed, we ack that there is no circle ci, but, there _is_ a clang-format test in place in Travis.  If you drill down through, you can see the result (clean):

> $ git diff -U0 --no-color HEAD^1 *.c *.cc *.cpp *.h *.hpp *.i *.ic *.ih | ./clang-format-diff.py -binary=clang-format-5.0 -style=file -p1 >_GIT_DIFF
> The command "git diff -U0 --no-color HEAD^1 *.c *.cc *.cpp *.h *.hpp *.i *.ic *.ih | ./clang-format-diff.py -binary=clang-format-5.0 -style=file -p1 >_GIT_DIFF" exited with 0.

With that, my guess is that the 5.6 innodb clang-format file is probably not exactly correct for the 5.6 innodb format specification.